From 4329eeedfdb9d021ab46edf5052b5e8587a2d344 Mon Sep 17 00:00:00 2001
From: thongsyho <thong.ho.px@rvc.renesas.com>
Date: Mon, 13 Jul 2020 16:25:16 +0700
Subject: [PATCH 098/504] can: rcar_can: add support for r8a7744

Add support of the device tree probing for the Renesas RZG1N r8a7744
CAN controllers.

Signed-off-by: thongsyho <thong.ho.px@rvc.renesas.com>
Signed-off-by: Kha Tran <kha.tran.px@rvc.renesas.com>
Signed-off-by: Quynh Nguyen <quynh.nguyen.xb@renesas.com>
---
 drivers/net/can/rcar_can.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/can/rcar_can.c b/drivers/net/can/rcar_can.c
index fbf0e3e..8807724 100644
--- a/drivers/net/can/rcar_can.c
+++ b/drivers/net/can/rcar_can.c
@@ -903,6 +903,7 @@ static int __maybe_unused rcar_can_resume(struct device *dev)
 static SIMPLE_DEV_PM_OPS(rcar_can_pm_ops, rcar_can_suspend, rcar_can_resume);
 
 static const struct of_device_id rcar_can_of_table[] __maybe_unused = {
+	{ .compatible = "renesas,can-r8a7744" },
 	{ .compatible = "renesas,can-r8a7745" },
 	{ .compatible = "renesas,can-r8a77470" },
 	{ .compatible = "renesas,can-r8a7778" },
-- 
2.7.4

